examples/jackd-talker/jackd_talker.c

```diff
@@ -766,7 +766,7 @@ int pci_connect()
 		}
 		printf("attaching to %s\n", devpath);
 		err = igb_attach(devpath, &igb_dev);
-		if (err) {
+		if (err || igb_attach_tx( &igb_dev )) {
```

Comment:
For me, it's also confusing. However, I noticed that it is done the same way for other examples like the `simple_talker.c` (line 379):

```c
err = igb_attach(devpath, igb_dev);
		if ( err || igb_attach_tx( igb_dev )) {
			printf("attach failed! (%s)\n", strerror(errno));
			continue;
		}
```